unicodetools/data/ucdxml/dev/ucd.nounihan.grouped.xml

```diff
@@ -31250,15 +31250,15 @@
          <char cp="109F4" na="MEROITIC CURSIVE NUMBER EIGHT HUNDRED THOUSAND" nv="800000"/>
          <char cp="109F5" na="MEROITIC CURSIVE NUMBER NINE HUNDRED THOUSAND" nv="900000"/>
          <char cp="109F6" na="MEROITIC CURSIVE FRACTION ONE TWELFTH" nv="1/12"/>
-         <char cp="109F7" na="MEROITIC CURSIVE FRACTION TWO TWELFTHS" nv="2/12"/>
-         <char cp="109F8" na="MEROITIC CURSIVE FRACTION THREE TWELFTHS" nv="3/12"/>
-         <char cp="109F9" na="MEROITIC CURSIVE FRACTION FOUR TWELFTHS" nv="4/12"/>
+         <char cp="109F7" na="MEROITIC CURSIVE FRACTION TWO TWELFTHS" nv="1/6"/>
```

Comment:
I thought that the previous change to the UnicodeData.txt fractions like 2/12 was an improvement. Can we not keep it that way now?

Reply:
Hmm, but John isn’t touching anything that would affect that…

I know why it went to 2/12: I changed Numeric_Value parsing in #1123. That got propagated to UCDXML when you regenerated in #1144.

OK, I am pretty sure this is #1125 again, stale BIN files with the old parsing of Numeric_Value. @jowilco, please delete your `Generated\BIN` directory and regenerate UCDXML.

Reply:
Hi @markusicu - regenerated with the Numeric_Value fix.

@eggrobin - thanks for the pointer!